src/coxph.jl

```diff
@@ -0,0 +1,88 @@
+function _cox_f(β, c::CoxAux{T})::T where T
+    grad = zeros(T, length(β))
+    hes = zeros(T, length(β), length(β))
+    _cox_fgh!(β, grad, hes, c::CoxAux{T}, false)
+end
+
+function _cox_fgh!(β, grad, hes, c::CoxAux{T}, compute_derivatives)::T where T
+    #get relevant quantities to compute negative loglikelihood, gradient and hessian
```

Comment:
It's more common to put comments about the whole function before it's signature.